src/main/java/executor/service/service/stepexecutors/selenium/StepExecutionClickCss.java

```diff
@@ -0,0 +1,13 @@
+package executor.service.service.stepexecutors.selenium;
+
+import executor.service.model.entity.Step;
+import executor.service.service.stepexecutors.StepExecution;
+import org.openqa.selenium.WebDriver;
+
+public interface StepExecutionClickCss extends StepExecution {
```

Comment:
why one interface extends another same interface?

Reply:
according to the conditiions there are three interfaces, I think it better to add higher level of abstraction and made common interface for these interfaces

Reply:
ok then remove interface StepExecutionClickCss and two others.
One higher level interface StepExecution and 3 implementations.

Reply:
StepExecution abstraction on StepExecutionClickCss and other interfaces, no on implementations, look on structure of abstractions in Collections, common interface and inheritors.

Reply:
what a purpose of having 3 same interfaces instead of 1?